packages/rum/README.md

```diff
@@ -57,6 +59,8 @@ datadogRum.init({
       service?: string,
       env?: string,
       version?: string,
+      enableTracing?: boolean,
+      tracingAllowedUrls?: RegExp[],
```

Comment:
I think having a callback function for this (ex: `shouldTraceUrl(url: string): boolean`) would be better than a regex array. It would let the user to implement any strategy they want.